runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
This section has been replaced by the `create` section in `command-line-interface.md`.  If you feel like the ideas which this section used to address are not covered there, can you suggest new wording for the new location?

Or do you feel like this section somehow applies generically without needing to talk about a command-line API.  If so, can you describe how standard streams and other file descriptors are being supplied here?